torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1799,9 +1809,12 @@ def test_server_process_global_profiler(self):
         outer_profile_rref.rpc_sync().__exit__(None, None, None)
 
         inner_events = rpc.rpc_sync(dst_worker_name, get_events_from_profile, (inner_profile_rref,))
-        self._assert_top_level_events(inner_events, ['aten::sub'])
+        expected_inner_events = ['aten::sub']
+        expected_outer_events = expected_inner_events + ['aten::add']
+
+        self._assert_top_level_events(inner_events, expected_inner_events)
         outer_events = rpc.rpc_sync(dst_worker_name, get_events_from_profile, (outer_profile_rref,))
-        self._assert_top_level_events(outer_events, ['aten::add', 'aten::sub'])
+        self._assert_top_level_events(outer_events, expected_outer_events)
```

Comment:
this is doing the same thing as the old code, right?

Reply:
Yes, this is just changing variable names, the real change for this test is to just log the discrepancy in events in case of a failure, which will give us more of an idea into what we need to debug if this is ever hit again.